pyintelowl/pyintelowl.py

```diff
@@ -49,6 +49,27 @@ def session(self) -> requests.Session:
 
         return self._session
 
+    def verify_and_filter(self, analyzers_requested) -> List[str]:
```

Comment:
I do not understand the purpose of this. The verification/filtering would anyways be done on the IntelOwl server, so aren't we just duplicating processing ?

Reply:
I thought there needs to be a way to prevent the user from running the misconfigured analyzers, like there is in the GUI